stats/opentelemetry/client_metrics.go

```diff
@@ -44,12 +44,15 @@ func (h *clientStatsHandler) initializeMetrics() {
 		return
 	}
 
-	meter := h.o.MetricsOptions.MeterProvider.Meter("grpc-go " + grpc.Version)
+	meter := h.o.MetricsOptions.MeterProvider.Meter("grpc-go ", otelmetric.WithInstrumentationVersion(grpc.Version))
```

Comment:
Delete this space in the string and the one below please.

Reply:
Done.